core/modules/util/EventThread.cc

```diff
@@ -42,6 +42,21 @@ namespace lsst {
 namespace qserv {
 namespace util {
 
+
+void CommandQueue::queCmd(std::vector<Command::Ptr> const& cmds) {
+    {
+        LOGS(_log, LOG_LVL_WARN, "&&& queCmd a");
+        std::lock_guard<std::mutex> lock(_mx);
+        LOGS(_log, LOG_LVL_WARN, "&&& queCmd b");
+        for (auto const& cmd:cmds) {
+            LOGS(_log, LOG_LVL_WARN, "&&& queCmd c");
```

Comment:
This may produce the hell of the messages for the shared scan queries. Does it really need to be the `WARN` level rather than (preferably) `TRACE`?